lib/chargebeex/action.ex

```diff
@@ -19,6 +19,30 @@ defmodule Chargebeex.Action do
     end
   end
 
+  def list_stream(resource, params \\ %{}) do
+    path = resource_base_path(resource)
+    next_offset = Map.get(params, "next_offset", 0)
+
+    Stream.unfold(next_offset, fn
+      nil ->
+        nil
+
+      0 ->
+        {:ok, _status_code, _headers, response} = Client.get(path, params)
```

Comment:
In the case of the rate limit error (or some other not successful response from chargebee) we would like to return this error rather than an error on pattern matching. Can you, please, handle this case and add a test for it as well?